2009-10-23  Thomas Cougnard  <thomas.cougnard@gmail.com>

	* readelf.c (dynamic_info): Correct size of array.

--- binutils/readelf.c	17 Aug 2009 08:34:37 -0000	1.454.2.4
+++ binutils/readelf.c	23 Oct 2009 00:15:41 -0000	1.454.2.5
@@ -165,7 +165,7 @@ static Elf_Internal_Syminfo * dynamic_sy
 static unsigned long dynamic_syminfo_offset;
 static unsigned int dynamic_syminfo_nent;
 static char program_interpreter[PATH_MAX];
-static bfd_vma dynamic_info[DT_JMPREL + 1];
+static bfd_vma dynamic_info[DT_ENCODING];
 static bfd_vma dynamic_info_DT_GNU_HASH;
 static bfd_vma version_info[16];
 static Elf_Internal_Ehdr elf_header;
